src/altair/networking.rs

```diff
@@ -0,0 +1,31 @@
+use crate::primitives::Epoch;
+use enr;
+pub use multiaddr::Multiaddr;
+#[cfg(feature = "serde")]
+use ssz_rs::prelude::Bitvector;
+use std::time::Duration;
+pub const ATTESTATION_SUBNET_COUNT: usize = 64;
+pub const SYNC_COMMITTEE_SUBNET_COUNT: usize = 4;
+pub const GOSSIP_MAX_SIZE: usize = 2usize.pow(20);
+pub const MAX_REQUEST_BLOCKS: usize = 2usize.pow(10);
+pub const MIN_EPOCHS_FOR_BLOCK_REQUESTS: Epoch = 33024;
+pub const MAX_CHUNK_SIZE: usize = 2usize.pow(20);
+pub const TTFB_TIMEOUT: Duration = Duration::from_secs(5);
+pub const RESP_TIMEOUT: Duration = Duration::from_secs(10);
+pub const ATTESTATION_PROPAGATION_SLOT_RANGE: usize = 32;
+pub const MAXIMUM_GOSSIP_CLOCK_DISPARITY: Duration = Duration::from_millis(500);
```

Comment:
imo these can all live in `phase0/networking.rs`

if it is nicer to expose them via `altair::networking` for some reason then we can just `pub use ...` here